website/public/js/controllers/settingsCtrl.js

```diff
@@ -66,12 +66,23 @@ habitrpg.controller('SettingsCtrl',
       User.set({'flags.newStuff':true});
     }
 
-    $scope.saveDayStart = function(){
-      var dayStart = +User.user.preferences.dayStart;
-      if (_.isNaN(dayStart) || dayStart < 0 || dayStart > 24) {
+    $scope.passDayStart = User.user.preferences.dayStart;
```

Comment:
If you look at my change to the jade module, you'll see I'm replacing a direct reference to user.preferences.dayStart with this variable and then passing it back as an argument to the function saveDayStart. I named it passDayStart to try to try to warn the reader this is a variable that's being passed around. I'd love to use a clearer name. 

In order to make it both appear as a default when loading the page in the first place and then be the value selected by the player when the code ran later, I discovered I had to send it back as an argument to the function, and give it a different name when it gets to the function.